### DISCUSSION TITLE
Help with reading an image/video file that is in a tmp path

### DISCUSSION THREAD

**Discussion by andresmejia3** [react-native-image-picker/react-native-image-picker#2143] [Q&A]
Hi, the way I understand it is that the image picker shows you thumbnails of the images/videos, when you select one, it copies the data from the actual image's URI on iOS to the `tmp` folder. Then it gives you the URI for the tmp folder. 

I get a path similar to this one 
`/Users/user/Library/Developer/CoreSimulator/Devices/0BA3C76A-9293-4F27-8A1A-4707C3A38D42/data/Containers/Data/Application/0135C34C-267D-4ABC-93BF-248A493AE2D2/tmp/74C108CF-575A-46C9-A37C-7234573F2AEB.jpg`

However, whenever I try to read the image at that path using`react-native-fetch-blob-util`'s `readStream`, I get an error saying `Error: No such file`. I'm assuming that's because the tmp file was already removed. Sometimes it successfully reads it, but 80% of the time it doesn't.

```

    const response = await launchImageLibrary({ mediaType: 'mixed' })
    const { uri, fileName, type } =  response.assets[0]
    const path = uri.split('//')[1] // "file://" needs to be stripped away from the URI for ReactNativeBlobUtil to work
    const data = await ReactNativeBlobUtil.fs.readFile(path, 'base64')

```

Any help would be appreciated, thank you!

**Comment by stempora** [comment]
Hello,

Did you find a solution for this issue ?

Thanks
